python/cudf/cudf/core/column/string.py

```diff
@@ -1113,15 +1062,22 @@ def binary_operator(self, binop, rhs, reflect=False):
         if reflect:
             lhs, rhs = rhs, lhs
         if isinstance(rhs, StringColumn) and binop == "add":
-            return lhs.nvstrings.cat(others=rhs.nvstrings)
+            return lhs.str().cat(others=rhs)
         else:
             msg = "{!r} operator not supported between {} and {}"
             raise TypeError(msg.format(binop, type(self), type(rhs)))
 
     def sum(self, dtype=None):
-        # dtype is irrelevant it is needed to be in sync with
-        # the sum method for Numeric Series
-        return self.nvstrings.join().to_host()[0]
+        # Should we be raising here? Pandas can't handle the mix of strings and
+        # None and throws, but we already have a test that looks to ignore
+        # nulls and returns anyway.
+
+        # if self.null_count > 0:
+        #     raise ValueError("Cannot get sum of string column with nulls")
```

Comment:
Unsure on the exact behavior we should have here with regards to nulls. @tanyamehrotra01 Do you have any opinions here since you did the initial implementation?

Reply:
We can update in a follow up PR if desired since this maintains current behavior